code/modules/organs/blood.dm

```diff
@@ -15,13 +16,13 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 	if(vessel)
 		return
 
-	vessel = new/datum/reagents(600)
+	vessel = new/datum/reagents(BLOOD_VOLUME_NORMAL + 50)
```

Comment:
600 != 560 + 50.

Reply:
Yeah it was buffed a little bit accidentally. I'll fix it back if it's an issue.